src/components/ToolIcon.scss

```diff
@@ -50,7 +60,7 @@
   font-size: 0.8em;
 }
 
-.ToolIcon_type_button {
+.excalidraw .ToolIcon_type_button, .ToolIcon_type_button {
```

Comment:
why the duplicated selector (separate from `.excalidraw)

Reply:
Earlier in `style.scss` there was no scoping was `button ` was  a global selector and `ToolIcon_type_button` was having higher precedence so this css was getting applied but now the css is scoped inside `excalidraw class in `style.scss` hence we need to override here.
Two selectors are needed because when some dialog example export dialog appears that time excalidraw class won't be present so `.ToolIcon_type_button ` this will work fine but otherwise, it needs to get overridden due to specificity.